workload: fix tpcc workers not work

louishust opened the pull request:
when run workload like below:
workload run tpcc  --warehouses 1 --workers 20 --wait=false
The workers parameter not work.

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/24440)
<!-- Reviewable:end -->

louishust committed:
workload: fix tpcc workers not work

when run workload like below:
workload run tpcc  --warehouses 1 --workers 20 --wait=false
The workers parameter not work.

jordanlewis commented:
@louishust what are you trying to accomplish here?

louishust commented:
@jordanlewis  when i run 
```
 workload run tpcc --warehouses 1 --workers 20 --wait=false 
```
I connect to crdb and execute `show sessions`, found the  concurrency is not 20 on crdb,
just 1 session run tpcc test. I think even warehouse is 1, the workers should be not only one.

jordanlewis commented:
TPCC mandates a specific number of workers per warehouse, which is why this is done this way. Sorry Louis (and sorry for letting this go incredibly stale).
